packages/cli/package-lock.json

```diff
@@ -4454,11 +4603,27 @@
 				"safe-buffer": "^5.1.1"
 			}
 		},
+		"run-async": {
+			"version": "2.3.0",
+			"resolved": "https://registry.npmjs.org/run-async/-/run-async-2.3.0.tgz",
+			"integrity": "sha1-A3GrSuC91yDUFm19/aZP96RFpsA=",
+			"requires": {
+				"is-promise": "^2.1.0"
+			}
+		},
 		"rustbn.js": {
 			"version": "0.1.2",
 			"resolved": "https://registry.npmjs.org/rustbn.js/-/rustbn.js-0.1.2.tgz",
 			"integrity": "sha512-bAkNqSHYdJdFsBC7Z11JgzYktL31HIpB2o70jZcGiL1U1TVtPyvaVhDrGWwS8uZtaqwW2k6NOPGZCqW/Dgh5Lg=="
 		},
+		"rxjs": {
```

Comment:
Damn, `rxjs` is heavy (17mb). Doesn't matter though, inquirer is beautiful and is totally worth it :-)